src/main/java/org/sonarsource/auth/github/GitHubIdentityProvider.java

```diff
@@ -85,7 +89,7 @@ public boolean allowsUsersToSignUp() {
   public void init(InitContext context) {
     String state = context.generateCsrfState();
     OAuthService scribe = prepareScribe(context)
-      .scope("user:email")
+      .scope("user:email,read:org")
```

Comment:
I was asking myself this question, but if the option is enabled then disabled then enabled, the user will have to validate each time new permissions.
But that's probably a corner case as it should never happen...